src/test/kotlin/coverosR3z/uitests/UITestHelpers.kt

```diff
@@ -47,22 +46,6 @@ fun startupTestForUIWithoutServer(domain: String = "", port : Int, driver: () ->
     return PageObjectModel.make(driver(), port, domain)
 }
 
-class EnterTimePage(private val driver: WebDriver, private val domain : String) {
-
-    fun enterTime(project: String, time: String, details: String, date: String) {
```

Comment:
this is only using view time - not even sure how it landed here.  Moved this to ViewTimePage